### PR TITLE
initialized structs and enum, created main

### PR DESCRIPTION
**Pull request by xahen**
can always be changed later, this is just a skeleton

### CODE REVIEW

File: src/package_distribution.c [hunk 1]
```diff
@@ -2,3 +2,4 @@
 #include <stdlib.h>
 
 #include "package_distribution.h"
+
```
Comment (oliverdalgaard) [inline src/package_distribution.c:5]:
```suggestion
```

Why whitespace bruhhh

**Review by oliverdalgaard — CHANGES_REQUESTED** [review]
You need to remove the whitespce

**Review by oliverdalgaard — APPROVED** [review]

File: src/package_distribution.c [hunk 2]
```diff
@@ -2,4 +2,3 @@
 #include <stdlib.h>
 
 #include "package_distribution.h"
```
Comment (oliverdalgaard) [inline src/package_distribution.c:4]:
```suggestion
#include "package_distribution.h"

```
Why no whitespace?

**Review by oliverdalgaard — APPROVED** [review]

**Review by kresten-j — APPROVED** [review]
Ser godt ud!

**Pull request merged**